src/ServiceControl.Plugin.CustomChecks/Internal/ICheck.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ServiceControl.Plugin.CustomChecks.Internal
+{
+    public interface ICheck
```

Comment:
I introduced it for the category and the id to cleanup the backend code. I
plan to go towards abstract classes only after the doco. Ok if I
address this later?

On Wednesday, January 27, 2016, Indu Alagarsamy <notifications@github.com>
wrote:

> In src/ServiceControl.Plugin.CustomChecks/Internal/ICheck.cs
> <https://github.com/Particular/ServiceControl.Plugin.Nsb6.CustomChecks/pull/2#discussion_r51025850>
> :
>
> > @@ -0,0 +1,8 @@
> > +﻿namespace ServiceControl.Plugin.CustomChecks.Internal
> > +{
> > +    public interface ICheck
>
> Seems like an unnecessary abstraction.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/ServiceControl.Plugin.Nsb6.CustomChecks/pull/2/files#r51025850>
> .
>